### ISSUE TITLE
Version 1.0.1 was published w/o lib directory

### ISSUE THREAD

**Issue by doronpr** [emilioicai/react-native-safe-modules#7]
Compare:
https://registry.npmjs.org/react-native-safe-modules/-/react-native-safe-modules-1.0.0.tgz
to
 https://registry.npmjs.org/react-native-safe-modules/-/react-native-safe-modules-1.0.1.tgz

can you please rollback or fix?

**Comment by Kos-M** [comment]
Damn  , this may affects lottie-react-native to run correct 
```
To reload the app press "r"
To open developer menu press "d"

info Reloading app...
 BUNDLE  ./index.js 

error: Error: While trying to resolve module `react-native-safe-modules` from file `/home/kosm/Documents/FreeSpot/reactnativeapp/node_modules/lottie-react-native/lib/LottieView.js`, the package `/home/kosm/Documents/FreeSpot/reactnativeapp/node_modules/react-native-safe-modules/package.json` was successfully found. However, this package itself specifies a `main` module field that could not be resolved (`/home/kosm/Documents/FreeSpot/reactnativeapp/node_modules/react-native-safe-modules/lib/index.js`. Indeed, none of these files exist:

  * /home/kosm/Documents/FreeSpot/reactnativeapp/node_modules/react-native-safe-modules/lib/index.js(.native|.android.js|.native.js|.js|.android.json|.native.json|.json|.android.ts|.native.ts|.ts|.android.tsx|.native.tsx|.tsx)
  * /home/kosm/Documents/FreeSpot/reactnativeapp/node_modules/react-native-safe-modules/lib/index.js/index(.native|.android.js|.native.js|.js|.android.json|.native.json|.json|.android.ts|.native.ts|.ts|.android.tsx|.native.tsx|.tsx)
    at DependencyGraph.resolveDependency (/home/kosm/Documents/FreeSpot/reactnativeapp/node_modules/metro/src/node-haste/DependencyGraph.js:436:17)
    at Object.resolve (/home/kosm/Documents/FreeSpot/reactnativeapp/node_modules/metro/src/lib/transformHelpers.js:317:42)
    at resolve (/home/kosm/Documents/FreeSpot/reactnativeapp/node_modules/metro/src/DeltaBundler/traverseDependencies.js:629:33)
    at /home/kosm/Documents/FreeSpot/reactnativeapp/node_modules/metro/src/DeltaBundler/traverseDependencies.js:645:26
    at Array.reduce (<anonymous>)
    at resolveDependencies (/home/kosm/Documents/FreeSpot/reactnativeapp/node_modules/metro/src/DeltaBundler/traverseDependencies.js:644:33)
    at /home/kosm/Documents/FreeSpot/reactnativeapp/node_modules/metro/src/DeltaBundler/traverseDependencies.js:329:33
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/home/kosm/Documents/FreeSpot/reactnativeapp/node_modules/metro/src/DeltaBundler/traverseDependencies.js:137:24)
    at _next (/home/kosm/Documents/FreeSpot/reactnativeapp/node_modules/metro/src/DeltaBundler/traverseDependencies.js:159:9)
```
Any tips how to force  lottie to use 1.0.0?

**Comment by doronpr** [comment]
This is fixed in 1.0.3. Closing.
